logic/src/commonMain/kotlin/com/wire/kalium/logic/sync/receiver/ConversationEventReceiver.kt

```diff
@@ -317,13 +324,19 @@ class ConversationEventReceiverImpl(
                 )
             }
 
-    private suspend fun handleDeletedConversation(event: Event.Conversation.DeletedConversation) =
-        conversationRepository.deleteConversation(event.conversationId)
+    private suspend fun handleDeletedConversation(event: Event.Conversation.DeletedConversation): Either<CoreFailure, Unit> {
+        val conversation = conversationRepository.observeConversationDetailsById(event.conversationId).onlyRight().first()
+        return conversationRepository.deleteConversation(event.conversationId)
```

Comment:
interesting thing is that we remove only the deleted conversation, but not messages from it. Maybe this is something that should be added too?

Reply:
Hey! Thanks for the comment, under the hood deleting the conversation will cascade all the "child" tables content related =)